### PR TITLE
Added support for VpnTunnelOptionsSpecifications

### PR DESCRIPTION
**Pull request by cigan1**
Support for VpnTunnelOptionsSpecifications allows specification of PreSharedKey and TunnelInsideCidr.

### CODE REVIEW

File: troposphere/ec2.py [hunk 1]
```diff
@@ -563,6 +563,13 @@ class VPCGatewayAttachment(AWSObject):
     }
 
 
+class VpnTunnelOptionsSpecification(AWSProperty):
+    props = {
+        'PreSharedKey': (basestring, False),
+        'TunnelInsideCidr': (basestring, False)
```
Comment (markpeek) [inline troposphere/ec2.py]: Best to add trailing commas.

File: troposphere/ec2.py [hunk 2]
```diff
@@ -572,6 +579,9 @@ class VPNConnection(AWSObject):
         'StaticRoutesOnly': (boolean, False),
         'Tags': ((Tags, list), False),
         'VpnGatewayId': (basestring, True),
+        'VpnTunnelOptionsSpecifications': (
+            [VpnTunnelOptionsSpecification], False
+        )
```
Comment (markpeek) [inline troposphere/ec2.py]: Best to add trailing commas.

File: troposphere/ec2.py [hunk 3]
```diff
@@ -563,6 +563,13 @@ class VPCGatewayAttachment(AWSObject):
     }
 
 
+class VpnTunnelOptionsSpecification(AWSProperty):
+    props = {
```
Comment (markpeek) [inline troposphere/ec2.py]: Would you want to add validators for these two properties as well?

PreSharedKey
Constraints: Allowed characters are alphanumeric characters and ._. Must be between 8 and 64 characters in length and cannot start with zero (0).

TunnelInsideCidr
Constraints: A size /30 CIDR block from the 169.254.0.0/16 range. The following CIDR blocks are reserved and cannot be used:

169.254.0.0/30
169.254.1.0/30
169.254.2.0/30
169.254.3.0/30
169.254.4.0/30
169.254.5.0/30
169.254.169.252/30

File: troposphere/ec2.py (hunk 1 above)
Comment (cigan1) [reply troposphere/ec2.py]: Added.

File: troposphere/ec2.py (hunk 2 above)
Comment (cigan1) [reply troposphere/ec2.py]: Added.

File: troposphere/ec2.py (hunk 3 above)
Comment (cigan1) [reply troposphere/ec2.py]: I will work on this.

**Comment by cigan1** [comment]
Added validator for TunnelInsideCidr.

File: troposphere/ec2.py (hunk 3 above)
Comment (cigan1) [reply troposphere/ec2.py]: @markpeek validator added.

**Pull request closed without merging**